bin/stacks/api-stack.ts

```diff
@@ -242,185 +244,185 @@ export class APIStack extends cdk.Stack {
+    //     allowOrigins: aws_apigateway.Cors.ALL_ORIGINS,
+    //     allowMethods: aws_apigateway.Cors.ALL_METHODS,
+    //   },
+    // });
+    // quote.addMethod('POST', quoteLambdaIntegration);
+
+    // const hardQuoteLambdaIntegration = new aws_apigateway.LambdaIntegration(hardQuoteLambdaAlias, {});
+    // const hardQuote = api.root.addResource('hard-quote', {
+    //   defaultCorsPreflightOptions: {
+    //     allowOrigins: aws_apigateway.Cors.ALL_ORIGINS,
+    //     allowMethods: aws_apigateway.Cors.ALL_METHODS,
+    //   },
+    // });
+    // hardQuote.addMethod('POST', hardQuoteLambdaIntegration);
+
+    // const switchLambdaIntegration = new aws_apigateway.LambdaIntegration(switchLambdaAlias, {});
+    // const switchResource = api.root.addResource('synthetic-switch', {
+    //   defaultCorsPreflightOptions: {
+    //     allowOrigins: aws_apigateway.Cors.ALL_ORIGINS,
+    //     allowMethods: aws_apigateway.Cors.ALL_METHODS,
+    //   },
+    // });
+    // const enabled = switchResource.addResource('enabled');
 
     /* add auth key */
-    const apiAuthzKey = api.addApiKey('AuthzKey');
-    const plan = api.addUsagePlan('AccessPlan', {
-      name: 'AccessPlan',
-    });
-    plan.addApiKey(apiAuthzKey);
-    plan.addApiStage({
-      stage: api.deploymentStage,
-    });
-
-    enabled.addMethod('GET', switchLambdaIntegration, { apiKeyRequired: true });
-
-    /*
-     * Param Dashboard Stack Initialization
-     */
-    new ParamDashboardStack(this, 'ParamDashboardStack', {
-      quoteLambda,
-    });
+    // const apiAuthzKey = api.addApiKey('AuthzKey');
+    // const plan = api.addUsagePlan('AccessPlan', {
+    //   name: 'AccessPlan',
+    // });
+    // plan.addApiKey(apiAuthzKey);
+    // plan.addApiStage({
+    //   stage: api.deploymentStage,
+    // });
+
+    // enabled.addMethod('GET', switchLambdaIntegration, { apiKeyRequired: true });
+
+    // /*
+    //  * Param Dashboard Stack Initialization
+    //  */
+    // new ParamDashboardStack(this, 'ParamDashboardStack', {
+    //   quoteLambda,
+    // });
 
     /*
      * Analytics Stack Initialization
      */
-    const analyticsStack = new AnalyticsStack(this, 'AnalyticsStack', {
-      quoteLambda,
-      hardQuoteLambda,
-      envVars: props.envVars,
-      analyticsStreamArn: firehoseStack.analyticsStreamArn,
-      stage,
-      chatbotSNSArn,
-    });
-
-    const cronStack = new CronStack(this, 'CronStack', {
-      RsDatabase: analyticsStack.dbName,
-      RsClusterIdentifier: analyticsStack.clusterId,
-      RedshiftCredSecretArn: analyticsStack.credSecretArn,
-      lambdaRole: lambdaRole,
-      chatbotSNSArn: chatbotSNSArn,
-      stage: stage,
-    });
-
-    new CronDashboardStack(this, 'CronDashboardStack', {
-      synthSwitchLambdaName: cronStack.synthSwitchCronLambda.functionName,
-      quoteLambdaName: quoteLambda.functionName,
-    });
+    // const analyticsStack = new AnalyticsStack(this, 'AnalyticsStack', {
```

Comment:
maybe instead of commenting out the whole stack, we simply use hard-coded logGroup name (so that we don't have to pass in quote/hardQuoteLambda?